api/cryptocom/ws_actions_test.go

```diff
@@ -0,0 +1,568 @@
+package cryptocom
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/bmizerany/assert"
+	"github.com/google/uuid"
+	"github.com/shopspring/decimal"
+)
+
+type mockRequest struct {
```

Comment:
Did not get it. Why this type exist? `Request` should be used in this cae

Reply:
Removed